src/main/java/org/rsa/listeners/SlashCommandListener.java

```diff
@@ -15,10 +19,19 @@ public void onSlashCommandInteraction(@NotNull SlashCommandInteractionEvent even
             return;
         }
 
-        try {
-            Commands.getCommand(event.getName()).handleSlashCommand(event);
-        } catch (ValidationException e) {
-            event.reply(e.toEventResponse()).setEphemeral(true).queue();
+        CommandObject command = Commands.getCommand(event.getName());
+        if (Objects.nonNull(command)) {
+            try {
+                command.handleSlashCommand(event);
+            } catch (ValidationException e) {
+                event.reply("Failed to process interaction.").setEphemeral(true).queue();
+            }
+            return;
+        }
+
+        CommandObjectV2 commandObjectV2 = Commands.getCommandV2(event.getName());
+        if (Objects.nonNull(commandObjectV2)) {
+            commandObjectV2.onSlashCommandInteraction(event);
         }
```

Comment:
Should add a check if the event has not been replied to and reply with a `No implementation found` message.